routes/health.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  path: '/health',
+  method: 'GET',
+  handler: (request, reply) => {
```

Comment:
is there a reason why there are no tags here - at least for api (+description) - Or is it just because it's basic basic at the moment